src/zope/viewlet/directives.rst

```diff
@@ -377,8 +376,7 @@ Neither the class or template have been specified:
   ... ''', context=context)
   Traceback (most recent call last):
   ...
-  ZopeXMLConfigurationError: File "<string>", line 3.2-7.8
-      ConfigurationError: Must specify a class or template
+  ConfigurationError: Must specify a class or template
```

Comment:
I was surprised to see that apparently we lost the location information. But when I ran the tests myself, I did see that it was still there, just moved down a line. I think the tests pass without it because of a doctest flag `IGNORE_EXCEPTION_DETAIL`, but I also think it's good to show that it's still reported. Would you mind adding it back?

Reply:
Of course not.